tools/site_and_regional/mesh_maker.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python3
+"""
+|------------------------------------------------------------------|
+|---------------------  Instructions  -----------------------------|
+|------------------------------------------------------------------|
+This script creates ESMF unstructured GRID (mesh file) from a netcdf
+file with valid lats and lons. Provided lats and lons can be 1D or 2D.
+
+For example for running WRF-CTSM cases, the user can create a mesh
+file for their domain :
+    ./mesh_maker.py --input wrfinput_d01 --output my_region
+        --lat XLAT --lon XLONG --verbose
+
+"""
```

Comment:
I'm not sure in what context this would be used. I suppose the idea is that this could be used in place of the two step process that relies on NCO and ESMF. This would enable you to create a mesh anywhere with a valid python environment rather than requiring the use (and build) of NCO and ESMF. Using those are also very machine specific.

A test for that would be to compare the results between the two methods.

Reply:
Comparing results to the two step process is probably a worthwhile thing to do. We had decided a long time ago that a python script with a conda environment would be an easier way to support this.